kube/conditions.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !plan9
+
+package kube
```

Comment:
I _think_ I put this into ./kube because it seemed logically better suited place than under ./cmd. But also deps is a good reason to not put it there- will try to move to a subpackage